test/integration/patients/worklist/schedule.js

```diff
@@ -81,32 +81,8 @@ context('schedule page', function() {
         });
 
         fx.included.push(
-          {
-            id: '1',
-            type: 'flows',
-            attributes: _.extend(_.sample(fxFlows), {
-              name: 'Complex Care Management',
-              id: '1',
-            }),
-          },
-          {
-            id: '1',
-            type: 'patients',
-            attributes: _.extend(_.sample(fxPatients), {
-              first_name: 'Test',
-              last_name: 'Patient',
-              id: '1',
-            }),
-          },
-          {
-            id: '2',
-            type: 'patients',
-            attributes: _.extend(_.sample(fxPatients), {
-              first_name: 'LongTest',
-              last_name: 'PatientName',
-              id: '1',
-            }),
-          },
+          getResource(_.extend(_.sample(fxPatients), { id: '1' }), 'patients', { id: '1', first_name: 'Test', last_name: 'Patient' }),
```

Comment:
`data.id: '1'` needs to be set on the fixture data before it's sent to the `getResource()`.

Which is the reason for the `_.extend(_.sample(fxPatients), { id: '1' })`

Reply:
Yeah I think that's a bad design on my part in the `getResource`.

We should revert the changes to `getResource` and just do:
```js
getResource(_.extend({}, _.sample(fxPatients), { id: '1', first_name: 'Test', last_name: 'Patient' }, 'patients');
```
then if there's any relationships it can be:
```js
const thingy = getResource(...., 'thingys');
thingy.relationships.foo = getRelationship(...);
```